optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/src/main/java/org/optaplanner/workbench/screens/domaineditor/client/handlers/planner/PlannerDomainHandler.java

```diff
@@ -43,7 +47,7 @@ public int getPriority() {
 
     @Override
     public DomainEditor getDomainEditor( boolean newInstance ) {
-        PlannerDomainEditor domainEditor = IOC.getBeanManager().lookupBean( PlannerDomainEditor.class ).newInstance();
+        PlannerDomainEditor domainEditor = plannerDomainEditorProvider.get();
         domainEditor.setHandler( this );
```

Comment:
Just a warning here. Since PlannerDomainHandler is ```@ApplicationScoped```, every domain editor created here will live forever. It won't be destroyed until the application itself closes.